pkgs/servers/sunshine/default.nix

```diff
@@ -121,21 +124,20 @@ stdenv.mkDerivation rec {
     libxcb
   ];
 
-  CXXFLAGS = [
-    "-Wno-format-security"
-  ];
-  CFLAGS = [
-    "-Wno-format-security"
-  ];
-
   cmakeFlags = [
     "-Wno-dev"
   ];
 
+  NIX_LDFLAGS = "-lX11";
```

Comment:
Can you clarify why this is needed?

This feels like it's an error in upstream's CMakeLists file or an unintended side-effect of ffmpeg.diff (or both honestly)

Reply:
Actually yeah if I add a line to the patch right after the ffmpeg one that does `list(APPEND PLATFORM_LIBRARIES ${X11_LIBRARIES})` it fixes it

That's an upstream problem since I see they are including `${X11_INCLUDE_DIRS}` when X11 is found but not actually adding the libraries like it does for other deps (and I suppose it worked by accident because of their precompiled ffmpeg binaries)

Reply:
Agree -- found a similar thing in testing but was trying to minimize scope of the patch to reduce complexity of future maintenance. Think it's better to move it to a patch or leave it as-is?

Reply:
Make a PR upstream and fetchpatch it is probably preferred

Reply:
If you'd like I can also make the PR. Just would be nice to not have to maintain any hack if we can. Thank you for working on this though!

Reply:
Opened upstream PR and pulled it in via fetchpatch for now.